neural_speed/models/llama/llama.cpp

```diff
@@ -82,6 +104,7 @@ static bool llama_model_eval_internal(model_context* ctx, const model_input* inp
   const int n_ctx = lctx.n_ctx;  // max number fo tokens to keep in the kv-cache
   const int n_keep = lctx.n_keep;
   const bool shift_roped_k = lctx.shift_roped_k;
+  MODEL_ASSERT(("continuous batching mechanism doesn't support shift rope.\n", !(concat_multi_seqs && shift_roped_k)));
```

Comment:
could we auto disable continuous batch when consumer enable shift rope rathe than assert false?

Reply:
add fallback in https://github.com/intel/neural-speed/pull/145/commits/b709ee965594eb8d70575d14069e69860b720241. But I think the assert should be kept as `continuous batching` could be modified by mistake.